zerver/tests/tests.py

```diff
@@ -201,19 +219,23 @@ def test_admin_api(self):
 class WorkerTest(TestCase):
     class FakeClient(object):
         def __init__(self):
-            self.consumers = {}
-            self.queue = []
+            # type: () -> None
+            self.consumers = {} # type: Dict[str, Callable]
+            self.queue = [] # type: List[Dict[str, Any]]
```

Comment:
(Also, let me know when you're done reviewing, since I know this is long enough it's possible you might review it in a few sittings)